src/patternfly/components/DataToolbar/data-toolbar.scss

```diff
@@ -0,0 +1,353 @@
+$pf-c-data-toolbar--breakpoint-map: build-breakpoint-map("base", "md", "lg", "xl", "2xl");
+$pf-c-data-toolbar--spacer-map: build-spacer-map("none", "sm", "md", "lg");
+
+// Data toolbar
+.pf-c-data-toolbar {
+  --pf-c-data-toolbar--BackgroundColor: var(--pf-global--BackgroundColor--100);
+  --pf-c-data-toolbar--GridGap: var(--pf-global--spacer--md);
+
+  // Expandable content
+  --pf-c-data-toolbar__expandable-content--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-data-toolbar__expandable-content--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-data-toolbar__expandable-content--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-data-toolbar__expandable-content--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-data-toolbar__expandable-content--lg--PaddingTop: 0;
+  --pf-c-data-toolbar__expandable-content--lg--PaddingRight: 0;
+  --pf-c-data-toolbar__expandable-content--lg--PaddingBottom: 0;
+  --pf-c-data-toolbar__expandable-content--lg--PaddingLeft: 0;
+  --pf-c-data-toolbar__expandable-content--ZIndex: var(--pf-global--ZIndex--xs);
```

Comment:
Looking at @christiemolloy's [zindex refactor](https://github.com/patternfly/patternfly-next/pull/1901), this component should have a z-index of `--pf-global--ZIndex--xs`. That said, we're going to end up with situations like this - I just added an expanded dropdown after the expanded toolbar in the DOM. I think this is a likely scenario in a table or datalist with an open kebab menu in the first row on mobile.

<img width="663" alt="Screen Shot 2019-08-16 at 1 33 37 PM" src="https://user-images.githubusercontent.com/35148959/63189864-973c9480-c02a-11e9-95b1-e927161d52c5.png">

I'm not sure how this behaves in react, but I think we should consider closing open menus when you open another menu. I know this happens with dropdowns or selects on the page, but not sure what happens if you have a mix of things like dropdowns, selects, toolbar, options menu, etc. WDYT?

Reply:
@mcoker Valid point, for sure. The React dropdown closes itself with tab exit, but not with VO exit, even when another menu is opened. I think 100% agree, open menus should close when another menu is open.